tasks/ewbf-equihash-cuda.yaml

```diff
@@ -0,0 +1,15 @@
+# docker image settings, required section
+container:
+  # Image name to start on worker, required param.
+  image: "sonm/zcash-cuda-ewfb@sha256:39ce0ce5005131ee817890ec4a5721fcff7ed22583d6586953a0d5b1fc642f5d"
```

Comment:
wrong image name - correct one is "sonm/equihash-cuda-ewbf@sha256:39ce0ce5005131ee817890ec4a5721fcff7ed22583d6586953a0d5b1fc642f5d"